CONTRIBUTING.md

```diff
@@ -228,45 +179,19 @@ cd patternfly-react
 git remote add upstream https://github.com/patternfly/patternfly-react.git
 ```
 
-2.  Create a branch:
+2. Create a branch:
 
 ```text
-$ git checkout -b my-branch -t upstream/master
+$ git checkout -b my-branch upstream/master
 ```
 
-3. Generate your Component
```

Comment:
Are we no longer supporting generator?  It is mentioned elsewhere in the docs.

Reply:
Code contribution does not require generating a component. We include information about Generators under `### Using Generators`.